### PR TITLE
Fix lint warnings

### PR DESCRIPTION
**Pull request by DonaldRG**
## Description of changes
Fix lint warnings

[CSDK-745]: https://verygoodsecurity.atlassian.net/browse/CSDK-745?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: vgscollect-cardio/src/main/java/com/verygoodsecurity/api/cardio/ScanActivity.kt [hunk 1]
```diff
@@ -58,10 +58,10 @@ class ScanActivity : BaseTransmitActivity() {
             .putExtra(CardIOActivity.EXTRA_KEEP_APPLICATION_THEME, keepApplicationTheme)
 
 
-        suppressManualEnter?.let {
+        suppressManualEnter.let {
             scanIntent.putExtra(CardIOActivity.EXTRA_SUPPRESS_MANUAL_ENTRY, it)
         }
-        suppressConfirmation?.let {
+        suppressConfirmation.let {
```
Comment (DmytroDm) [inline vgscollect-cardio/src/main/java/com/verygoodsecurity/api/cardio/ScanActivity.kt]: @DonaldRG If `suppressConfirmation` is not nullable we can remove `let`.

File: vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/card/conection/InputCardNumberConnection.kt [hunk 2]
```diff
@@ -12,7 +12,7 @@ import com.verygoodsecurity.vgscollect.view.card.validation.LengthMatchValidator
 internal class InputCardNumberConnection(
     id: Int,
     validator: CompositeValidator,
-    private val IcardBrand: IDrawCardBrand? = null,
+    private val iCardBrand: IDrawCardBrand? = null,
```
Comment (DmytroDm) [inline vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/card/conection/InputCardNumberConnection.kt]: @DonaldRG Can we remove `i` and name this val just `cardBrand`?

File: app/src/main/java/com/verygoodsecurity/demoapp/VGSSampleApplication.kt [hunk 3]
```diff
@@ -1,13 +1,5 @@
...
-class VGSSampleApplication: Application() {
-
-    override fun onCreate() {
-        super.onCreate()
-//        MicroblinkSDK.setLicenseKey("<key>", this)
-//        MicroblinkSDK.setLicenseKey("<path>", this)
-    }
-}
\ No newline at end of file
+class VGSSampleApplication: Application()
```
Comment (DmytroDm) [inline app/src/main/java/com/verygoodsecurity/demoapp/VGSSampleApplication.kt]: @DonaldRG I think this is used for MicroblinkSDK demo example, please suppress the lint check and leave it as is.

File: vgscollect-cardio/src/main/java/com/verygoodsecurity/api/cardio/ScanActivity.kt (hunk 1 above)
Comment (DonaldRG) [reply vgscollect-cardio/src/main/java/com/verygoodsecurity/api/cardio/ScanActivity.kt]: Yup, that make sense, and same for `suppressManualEnter`

File: vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/card/conection/InputCardNumberConnection.kt (hunk 2 above)
Comment (DonaldRG) [reply vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/card/conection/InputCardNumberConnection.kt]: That make sense 👍

File: app/src/main/java/com/verygoodsecurity/demoapp/VGSSampleApplication.kt (hunk 3 above)
Comment (DonaldRG) [reply app/src/main/java/com/verygoodsecurity/demoapp/VGSSampleApplication.kt]: Okis!

**Review by DmytroDm — APPROVED** [review]

**Pull request merged**